scripts/inner/kubestellar-kube-bind

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# SM_CONFIG should be set before calling this script
```

Comment:
In that case it should be documented in `commands.md`.

Can we simplify and just suppose that `kubectl` without any tweaking will manipulate the SMAS?